integration-tests/bats/blame.bats

```diff
@@ -94,6 +96,8 @@ SQL
 }
 
 @test "blame: works with HEAD~1 as the commit ref" {
+    skip "SQL views do no support AS OF queries"
```

Comment:
Yeah they do, why doesn't this work?

Reply:
I narrowed the problem down to `AS OF` not being supported with Dolt system table:
```
blame> select * from dolt_blame_test as of 'head~1';
table not found: dolt_log
blame> select * from dolt_log;
+----------------------------------+-----------+------------------+-------------------------+-----------------------------------------------+
| commit_hash                      | committer | email            | date                    | message                                       |
+----------------------------------+-----------+------------------+-------------------------+-----------------------------------------------+
| kpgesuj5ti1t37qivv4j6n0e23a9kopu | andy      | andy@dolthub.com | 2022-06-22 20:29:05.992 | deleted                                       |
| 0chsp68li9hv9ejke9shbg1ki2ppn684 | andy      | andy@dolthub.com | 2022-06-22 20:19:17.671 | inserted again                                |
| dimr7rdd02nsbuph0jp3c310hlcrhhee | andy      | andy@dolthub.com | 2022-06-22 20:19:01.294 | inserted                                      |
| 500ei9911iins2jrlce2pca4d6ltmhnq | andy      | andy@dolthub.com | 2022-06-22 20:18:23.214 | added table test with two primary key columns |
| eflcel0p8pip61m33nibdtj9suohdd0v | andy      | andy@dolthub.com | 2022-06-22 20:14:03.972 | Initialize data repository                    |
+----------------------------------+-----------+------------------+-------------------------+-----------------------------------------------+
blame> select * from dolt_log as of 'head~1';
table not found: dolt_log
```
will fix